src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write code here
+[...document.querySelectorAll('li')].forEach(item => {
```

Comment:
FYI: HTML Collection (the result of `querySelectorAll` method) has built-it `forEach` method